src/routes/settings.route.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import authMiddleware from '../middlewares/auth.middleware';
+import SettingsController from '../controllers/settings.controller';
+import { validateUserRole } from '../validations/role.validation';
+
+const router = express.Router();
+
+router.patch('/reset-roles', authMiddleware.checkUserLoggedIn, validateUserRole, authMiddleware.verifyIfUserIsAdmin, SettingsController.changeUserRole);
```

Comment:
You can update this to `/roles` instead.

Reply:
finished now